packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/lv1-lv2-migration-banner/lv1-lv2-migration-banner.controller.js

```diff
@@ -0,0 +1,58 @@
+import {
+  GUIDE_LINKS,
+  LV1_LV2_REGEX,
+} from './lv1-lv2-migration-banner.constants';
+
+export default class DedicatedCloudLv1Lv2MigrationBanner {
+  /* @ngInject */
+  constructor($http, $q, coreConfig, iceberg) {
+    this.$http = $http;
+    this.$q = $q;
+    this.coreConfig = coreConfig;
+    this.iceberg = iceberg;
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.GUIDE_LINK =
+      GUIDE_LINKS[this.coreConfig.getUserLanguage().toUpperCase()] ||
+      GUIDE_LINKS.EN;
```

Comment:
related to this : https://github.com/ovh/manager/pull/9239/files#r1206697221

```suggestion
      GUIDE_LINKS[this.coreConfig.getUserLanguage().toUpperCase()] ||
      GUIDE_LINKS.DEFAULT;
```